templates/nextjs/src/app/page.tsx

```diff
@@ -59,18 +75,36 @@ export default function Home() {
         </span>,
       );
     }
+    try {
+      setIsLoading(true);
+      // Call the increment_counter function on the contract
+      const { waitForResult } = await contract.functions
+        .increment_counter(bn(1))
+        .call();
+
+      // Wait for the transaction to be mined, and then read the value returned
+      const { value, transactionId } = await waitForResult();
+      toast.success(() => (
+        <span>
+          Counter Incremented! View it on the
+          <a
+            target="_blank"
+            className="pl-1 underline"
+            href={`https://app.fuel.network/tx/${transactionId}`}
```

Comment:
Should this be conditional so it's only when the app is running against testnet?